src/__stories__/index.stories.js

```diff
@@ -209,4 +209,26 @@ storiesOf('Loads', module)
         )}
       </Loads>
     );
+  })
+  .add('load on mount (context cache)', () => {
```

Comment:
Remove this? Unless you got it working with storybook after our last discussion.

Reply:
Good point